tests/integration_tests/scheduler/test_lsf_driver.py

```diff
@@ -105,3 +105,30 @@ async def test_job_name():
     )
     stdout, _ = await bjobs_process.communicate()
     assert "my_job_name" in stdout.decode()
+
+
+@pytest.mark.parametrize(
+    "actual_returncode, returncode_that_ert_sees",
+    [
+        ([0, 0]),
+        ([1, 1]),
+        ([2, 1]),
+        ([255, 1]),
+        ([256, 0]),  # return codes are 8 bit.
```

Comment:
Is this one for the sake of overflowing?

Reply:
yup